Gruntfile.js

```diff
@@ -278,7 +282,7 @@ module.exports = function(grunt) {
                 helpers: 'tests/beforeEachSpec.js'
             },
             test: {
-                src: ['dist/*.js'],
+                src: ['dist/d3fc.min.js'],
```

Comment:
Not strictly necessary for this change but stops the minified and non-minified code being included (which is just weird).